### ISSUE TITLE
Cannot read property "indexOf" null

### ISSUE THREAD

**Issue by lauraemilyjacobsen** [digitalmedia34/updateconfig#8]
I was getting this error, the same as in #1 .

```

2019-02-26T05:27:05.6187481Z Task         : Update config
2019-02-26T05:27:05.6187672Z Description  : Update config variables in files
2019-02-26T05:27:05.6187835Z Version      : 1.2.2
2019-02-26T05:27:05.6187985Z Author       : Digitalmedia34
2019-02-26T05:27:05.6188204Z Help         : v1.2.2 - [More Information](https://github.com/digitalmedia34/updateconfig#readme)
2019-02-26T05:27:05.6188432Z ==============================================================================
2019-02-26T05:27:05.9355690Z ##[error]Cannot read property 'indexOf' of null
```

The cause was that one app setting in my config file did not have a matching release variable. Does this task require a variable to be present for every setting in the config file?

It would be great if this task would 1) provide a better error message or 2) even better, not require there to be a release variable for every config setting.

**Comment by lauraemilyjacobsen** [comment]
I believe this is related to #2 . 

My config file has two settings:

ToAddress
TestToAddresses

TestToAddresses was not defined in the release stage where I got this error. When I added a variable named TestToAddresses to this stage, the task succeeded.

I would like to install version 1.2.3 to see if this resolves the issue but like #3 , I'm also unable to install this version. I uninstalled and reinstalled but it still looks like I have version 1.2.2, at least that's what's printing in the logs.

It's odd because I see that the version is 1.2.3 in the marketplace but that's not what I'm getting.
